it/code_editor/instructions.md

```diff
@@ -12,11 +12,11 @@ Gedit è un editor open-source e gratuito, disponibile per tutti i sistemi opera
 
 Sublime Text è uno tra gli editor più utilizzati. Ha un periodo di prova gratuito. È molto facile da installare e da utilizzare ed è disponibile per tutti i sistemi operativi.
 
-[Scaricalo qui](https://www.sublimetext.com/2)
+[Scaricalo qui](http://www.sublimetext.com/2)
```

Comment:
Hi, why did you change the link to be http only here and in other places? It's always better to use https :)